modulefiles/tasks/jet/run_graphics.local.lua

```diff
@@ -0,0 +1,5 @@
+prepend_path("MODULEPATH","/contrib/miniconda3/modulefiles")
+load(pathJoin("miniconda3", os.getenv("miniconda3_ver") or "4.5.12"))
+
+setenv("SRW_ENV", "pygraf")
+
```

Comment:
@danielabdi-noaa - 
  Are there additional packages for `pygraf` environment in miniconda3/4.5.12 required for plotting  that are missing in current installation of `regional_workflow` environment under epic account in miniconda3/4.12.0? It is installed under /scratch1/NCEPDEV/nems/role.epic/miniconda3/modulefiles/. The plotting scripts were successfully generating test plots using this` regional_workflow` conda environment. That would be best to know what else needs to be installed and to update the environment on Hera and other platforms

Reply:
@natalie-perlin Yes, I've actually opened up an issue for this #561. It would be great to have the additional libraries such as `xarrays` that are needed for pygraf plotting besides ones needed for `plot_allvars` we have in SRW. I used the old modulefile "4.5.12" because there was no other alternative.